packages/tocco-ui/src/Icon/Icon.spec.js

```diff
@@ -32,7 +32,7 @@ describe('tocco-ui', function() {
         onMouseDown,
         position
       } = props
-      expect(Object.keys(props)).to.have.lengthOf(7)
+      expect(Object.keys(props)).to.have.lengthOf(8) // children is eighth prop
```

Comment:
what is the value of this comment? or even the line. to test for an amount of props does not seam to test anything?

Reply:
It should be passed 7 props but it is tested for 8 props. To clarify what makes the difference I add a comment. "props.children" is added by react what has confused me. I explain the pitfall to a former me or a forgetting me.

Reply:
please remove this line entirely. its confusing me more than there is a value to it

Reply:
ok